src/components/shared/LanguageSelector/LanguageSelector.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const LanguageSelector = ({ lang }) => {
```

Comment:
We could inject current lang in place of receiving it as a parameter. But I am not sure if it is a good idea, @fcoronelmakingsense, @damianmuti  what do you think?

```
/**
 * LanguageSelector
 * @param { Object } props
 * @param { import('react-intl').InjectedIntl } props.intl
 */
const LanguageSelector = ({ intl }) => {
  const lang = intl.locale;
  return (
    <div className="language-selector">
    // . . .
    </div>
  );
};
export default injectIntl(LanguageSelector);
```

Reply:
always has the current language, and that is how we use in other pages. maybe is better this

Reply:
This was my first thought when I've created this component, but my React knowledge isn't the best tho, so I couldn't get around it.
Thanks for the help, @andresmoschini ! I got it working this way, and I guess it's a good idea not to not have the locale as a parameter. 
@fcoronelmakingsense do you agree? If it's ok with you guys, I'll push the changes right away.

Reply:
🚀

Reply:
I'll update this PR right away 👍